Library/Config.ts

```diff
@@ -64,10 +58,28 @@ class Config {
     private _profileQueryEndpoint: string;
     /** Host name for quickpulse service */
     private _quickPulseHost: string;
-
+    
+    /** private config object that is populated from config json file */
+    private _config : ICustomConfig = CustomConfig.generateConfigurationObject();
+
+    public enableAutoCollectConsole: boolean; // enable auto collect console
```

Comment:
We need to have all of these new settings listed in readme, also it will be good to have a new section describing how to use JSON configuration file

Reply:
Readme added